src/site/stages/build/plugins/check-broken-links.js

```diff
@@ -50,7 +50,9 @@ function checkBrokenLinks(buildOptions) {
     const brokenLinkChecker = createBrokenLinkChecker({
       allowRedirects: true,
       warn:
-        buildOptions.watch || buildOptions.buildtype === ENVIRONMENTS.VAGOVDEV,
+        buildOptions.watch ||
+        buildOptions.buildtype === ENVIRONMENTS.VAGOVDEV ||
+        buildOptions.buildtype === ENVIRONMENTS.VAGOVSTAGING,
```

Comment:
`staging.va.gov` now points to CMS Staging, so those broken-link errors were resurfacing. We have to ignore them in Staging then too.